src/movies.js

```diff
@@ -1,16 +1,113 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
+function getAllDirectors(array) {
+    // array is the array of movies
+    // your code in here!
+    // return a value
+    let directorsArr = array.map((movie) => {
+        return movie.director
+    })
+    return directorsArr
+  }
+
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
+function howManyMovies(array) {
+
+let filterArray=array.filter ((movie)=>{
+    return movie.director === "Steven Spielberg" && movie.genre.includes("Drama")
```

Comment:
very good job putting both conditions in the same line and even using .includes!